src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" class="resets">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./blocks_css/general.css">
+  </head>
+  <body>
+
+    <div class="card" data-qa="card">
+      <img
+        class="card__picture"
+        src="./blocks_css/images/imac.jpeg"
+        alt="Apple A1419"
+      >
+
+      <h2 class="card__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__reviews-container">
+
+        <div class="stars stars--4">
```

Comment:
You only need empty lines between multiple sibling blocks, not between parent block and child (checklist p.3):
<img width="500" alt="image" src="https://user-images.githubusercontent.com/118751034/214652924-78d84a9f-b355-47d0-9ee3-fa141ba5f5ba.png">


```suggestion
      <div class="card__reviews-container">
        <div class="stars stars--4">
```